v4/source/dataretention.yaml

```diff
@@ -1,25 +1,500 @@
   /data_retention/policy:
     get:
       tags:
-        - dataretention
-      summary: Get the data retention policy details.
-      description: >
-        Gets the current data retention policy details from the server,
+        - data retention
+      summary: Get the global data retention policy
+      description: |
+        Gets the current global data retention policy details from the server,
         including what data should be purged and the cutoff times for each data
         type that should be purged.
-
+        
         __Minimum server version__: 4.3
+        
+        ##### Permissions
+        Requires an active session but no other permissions.
```

Comment:
I think we should put this and the other retention policies behind a specific permission.

Reply:
I set it to the MANAGE_SYSTEM permission for now.

Reply:
It should probably be more specific `manage_data_retention`. I missed that in the server PR.